### PR TITLE
Update sector list

### PR DESCRIPTION
**Pull request by marcuspp**
This PR updates the sector list so should not be merged till we are ready to do so.

It contains all changes to the sector list

### CODE REVIEW

File: mi/migrations/0003_sync_sectors.py [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+    (1, "Advanced engineering"),
+    (2, "Aerospace"),
+    (3, "Aerospace : Aircraft design"),
+    (4, "Aerospace : Component manufacturing"),
+    (5, "Aerospace : Component manufacturing : Engines"),
+    (7, "Aerospace : Maintenance"),
+    (8, "Aerospace : Manufacturing and assembly"),
+    (9, "Aerospace : Manufacturing and assembly : Aircraft"),
+    (10, "Aerospace : Manufacturing and assembly : Helicopters"),
+    (11, "Aerospace : Component manufacturing : Space technology"),
```
Comment (kristof-bb) [inline mi/migrations/0003_sync_sectors.py:20]: (11, "Aerospace : Component manufacturing : Space technology"),

File: mi/migrations/0003_sync_sectors.py [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+    (103, "Creative industries : Antiques and antiquities"),
+    (105, "Creative industries : Books, printed media and stationery"),
+    (107, "Creative industries : Multimedia sales"),
+    (110, "Creative industries : Experience economy"),
+    (112, "Creative industries : Advertising"),
+    (113, "Creative industries : Animation"),
+    (114, "Creative industries : Music"),
+    (115, "Creative industries : Publishing"),
+    (116, "Creative industries : Film and tv"),
+    (117, "Creative industries : Video games"),
```
Comment (kristof-bb) [inline mi/migrations/0003_sync_sectors.py:84]: (117, "Creative industries : Games"),

File: mi/migrations/0003_sync_sectors.py [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+    (314, "Creative industries : Digital media"),
+    (315, "Creative industries : Experience economy : Art"),
+    (316, "Creative industries : Experience economy : Attractions"),
+    (317, "Creative industries : Experience economy : Museums"),
+    (318, "Creative industries : Film and tv : Broadcasting"),
+    (319, "Creative industries : Film and tv : Post production"),
+    (320, "Creative industries : Film and tv : Production and finance"),
+    (321, "Creative industries : Games : Development"),
+    (322, "Creative industries : Games : Esports"),
+    (322, "Creative industries : Games : Publishing"),
```
Comment (kristof-bb) [inline mi/migrations/0003_sync_sectors.py:236]: 323

File: mi/migrations/0003_sync_sectors.py [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+    (444, "Technology and smart cities : Hardware"),
+    (445, "Technology and smart cities : Hardware : 5G"),
+    (446, "Technology and smart cities : Hardware : Data centres"),
+    (447, "Technology and smart cities : Hardware : Internet of things"),
+    (448, "Technology and smart cities : Hardware : Photonics"),
+    (449, "Technology and smart cities : Hardware : Semiconductors"),
+    (450, "Technology and smart cities : Hardware : Sensors"),
+    (451, "Technology and smart cities : Hardware : Telecommunications and electronics systems"),
+    (452, "Technology and smart cities : Software"),
+    (453, "Technology and smart cities : Software : 5G"),
```
Comment (kristof-bb) [inline mi/migrations/0003_sync_sectors.py:375]: do we use 5g or 5G?

**Comment by dvoong** [comment]
looks good

**Pull request merged**